index.html

```diff
@@ -0,0 +1,273 @@
+                <div class="connect">
+                    <div id="title">LET’S CONNECT</div>
+                    <ul class="social d-flex justify-start no-pad">
+                    <li>
+                            <a href="https://twitter.com/abiola_arobieke">
+                                <img src="./images/twitter-icon.png" alt="twitter-icon">
+                            <a/> 
+                    </li>
+                    <li>
+                            <a href="https://linkedin.com/in/abiola-arobieke">
+                                <img src="./images/linkedin-icon.png" alt="linkedin-icon">
+                            <a/> 
+                        </li>
+                    <li><a href="#"><img src="./images/medium-icon.png" alt="medium-icon"><a/> </li>
+                    <li> 
+                            <a href="https://github.com/abiola-arobieke">
+                                <img src="./images/github-icon.png" alt="github-icon">
+                            <a/> 
+                        </li>
+                    <li><a href="#"><img src="./images/hand-icon.png" alt="hand-icon"><a/> </li>
+                    </ul>
+                </div>
+                <button class="btn-action" type="button">Get my resume</button> 
+            </div>
+            <div class="skill-set">
+                <div class="tech lang d-flex justify-between">
+                    <h4>Frameworks</h4>
+                    <button type="button"> v </button>
+                </div>    
+                <ul class="d-flex flex-column no-pad">
+                    <li class="d-flex">
+                        <img src="./images/js-logo.png" alt="js-logo">
+                        <span>javascript</span>
+                    </li>
+                    <li class="d-flex">
+                        <img src="./images/html-logo.png" alt="html-logo"> 
+                        <span>HTML</span>
+                    </li>
+                    <li class="d-flex">
+                        <img src="./images/css-logo.png" alt="css-logo">
+                        <span>CSS</span>
+                    </li>
+                </ul>
+                <div>
+
+                    <div>
+                        <div class="tech d-flex justify-between">
+                            <h4>Frameworks</h4>
+                            <button type="button"> > </button>
+                        </div>
+                    </div>
+
+                    <div>
+                        <div class="tech d-flex justify-between">
+                            <h4>Skills</h4>
+                            <button type="button"> > </button>
+                        </div>
+                    </div>
+
+                </div>   
+            </div> 
+        </section>
+
+        <!-- contact section -->
+        <section class="contact-me contact-container d-flex flex-column ">
+            <div class="text-center mb-48">
+                <h1 id="contact-me">Contact me</h1>
```

Comment:
- [ ]  Make reference on lin `97` on a detailed review on why you need just one h1 element in a project 👍